content/blog/2016/2016-08-29-sauce-pipeline.adoc

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: "Browser-testing with Sauce OnDemand and Pipeline"
+tags:
+- pipeline
+- plugins
+- saucelabs
+- selenium
+- nightwatch
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at Cloudbees.
+
+Testing web applications across multiple browsers on different platforms can be challenging even for smaller applications.
+With Jenkins and the
+link:https://wiki.jenkins-ci.org/display/JENKINS/Sauce+OnDemand+Plugin[Sauce OnDemand Plugin],
+you can wrangle that complexity by defining your Pipeline as Code.
+
+
+=== Pipeline &#9829; UI Testing, Too
+I recently started looking for a way to do browser UI testing for an open-source JavaScript project.
```

Comment:
I think you should tweak this a little more to say "for an open-source JavaScript project to which I contribute" (or something like that).

That way the "but we're committed" makes more sense, since it will be more clear who the "we" is.